pjlib/include/pj/string.h

```diff
@@ -262,6 +262,21 @@ PJ_IDECL(pj_str_t*) pj_strdup2_with_null(pj_pool_t *pool,
 PJ_IDECL(pj_str_t) pj_strdup3(pj_pool_t *pool, const char *src);
 
 /**
+ * Duplicate string.
+ *
+ * @param pool	    The pool.
+ * @param dst	    The string result.
+ * @param src	    The string to duplicate.
+ * @param len	    The length of the string to duplicate.
+ *
+ * @return the string result.
+ */
+PJ_IDECL(pj_str_t*) pj_strdup4(pj_pool_t *pool,
```

Comment:
I duplicated the layout followed for the other strdup functions, yes. I think it's because of pjsip's mixed use of tabs and spaces